recipes/Time_Series/Bike_Sharing_Finetuning_with_Exogenous.ipynb

```diff
@@ -0,0 +1,1658 @@
+      "text/plain": [
+       "<Figure size 2000x300 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for target_column in target_columns:\n",
+    "    input_df.iloc[1000 : 1000 + 24 * 10].plot(\n",
+    "        x=timestamp_column, y=target_column, figsize=(20, 3)\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Zero shot forecasting\n",
+    "\n",
+    "In the first part of this notebook, we focus on zero-shot inference of the targets only. This does not consider the relationships between the targets or any of the additional exogenous features in the dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Split the data into training, validation, and testing sets\n",
+    "\n",
+    "We split the data into training, validation, and test sets. The training set is used to train the preprocessor, while the test set is used to evaluate the model performance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Data lengths: train = 8689, val = 4858, test = 4856\n"
+     ]
+    }
+   ],
+   "source": [
+    "split_config = {\"train\": 0.5, \"test\": 0.25}\n",
```

Comment:
woah does this work? my biggest pain in the tail when working throughthese examples was getting the split_config set right. And when I tried it like this, it complained that it couldn't index on a float (ie it expected a tuple or a list here).

Reply:
i'm truly asking if it works, because i saw docs that said it should, but in practice for me, it did not.

Reply:
@rawkintrevo I just reran the whole notebook in a clean env (locally) and had no issue with the split config. If you are getting an issue it would be good to figure out what is going on. Do you have version 0.2.10 installed?

Reply:
In the interest of not polluting this PR, let's take this to DM on the discord or slack or somewhere else. If you say it works, that's good enough for me.